shared/menubar/index.render.desktop.js

```diff
@@ -409,6 +410,13 @@ const styles = {
     position: 'absolute',
     right: 0,
     top: 5
+  },
+  dz1InputHint: {
+    color: globalColors.black,
+    // This is because normal <Input/> does center-aligned hints by using
+    // width: 100% on position: absolute
+    width: '',
```

Comment:
Does that actually work? what happens if you use the default value 'auto' (from [here](https://developer.mozilla.org/en-US/docs/Web/CSS/width))